### PR TITLE
Public repository information added

### PR DESCRIPTION
**Pull request by DrTorte**
Added note indicating that public repositories have to be enabled manually.

### CODE REVIEW

File: user/running-build-in-debug-mode.md [hunk 1]
```diff
@@ -11,10 +11,18 @@ significant differences between your local development environment and
 the Travis CI build environment, you can restart builds in debug mode
 to get shell access to the virtual machine or container.
 
+## Enabling debug mode
+
+Private repositories have debug mode enabled by default, and no changes need to be made.
+For public repositories, we have to enable it on a repository basis. Please contact us
+via email to support@travis-ci.com with the repository you want it active for, and we can
+enable it.
```
Comment (plaindocs) [inline user/running-build-in-debug-mode.md]: How about 

> To enable debug for your public repositories, please email us at
> [support@travis-ci.com](mailto:support@travis-ci.com) and let us know which repositories you want activated.

File: user/running-build-in-debug-mode.md [hunk 2]
```diff
@@ -11,10 +11,19 @@ significant differences between your local development environment and
...
+For public repositories, we have to enable it on a repository basis.  
+To enable debug for your public repositories, please email us at
+support@travis-ci.com and let us know which repositories you want activated.
+
 ## Restarting a job in debug mode
 
 The "Debug build" or "Debug job" button is available on the upper right corner of
-the build and job pages.
+the build and job pages for private repositories. For open source repositories, 
+this button is not available.
```
Comment (cotsog) [inline user/running-build-in-debug-mode.md]: Maybe mention that, in this case, the debug build needs to be started via an API call?

**Comment by DrTorte** [comment]
Ack, indeed it does look like I did! Let me rectify that right now. :)

**Comment by DrTorte** [comment]
Okay, hang on. This line is what you're referring to I believe?

"To enable debug for your public repositories, please email us at support@travis-ci.com and let us know which repositories you want activated."

Or did you mean you wanted something else there instead?

**Comment by plaindocs** [comment]
Yeah, I'm suggesting using that instead of 

For public repositories, we have to enable it on a repository basis.  
+To enable debug for your public repositories, please email us at
+support@travis-ci.com and let us know which repositories you want activated.

**Pull request merged**